src/native/fs/fs_napi.cpp

```diff
@@ -991,6 +995,28 @@ struct FileSetxattr : public FSWrapWorker<FileWrap>
     }
 };
 
+struct LinkFileAt : public FSWrapWorker<FileWrap>
+{
+    std::string _filepath;
+    int _rc_res;
+
+    LinkFileAt(const Napi::CallbackInfo& info)
+        : FSWrapWorker<FileWrap>(info)
+    {
+        _filepath = info[2].As<Napi::String>();
+        Begin(XSTR() << "LinkFileAt " << DVAL(_wrap->_path) << DVAL(_wrap->_fd) << DVAL(_filepath));
+    }
+    virtual void Work()
+    {
+        // Link the Linux linkat() if there is no old path use empty string.
+        _rc_res = gpfs_linkat(_wrap->_fd, "", AT_FDCWD, _filepath.c_str(), AT_EMPTY_PATH);
```

Comment:
_rc_res can be a local variable.

Reply:
done

Reply:
I suggest to keep it like all the other code, and not name a local variable with underscore -
```
int r = gpfs_linkat(...);
if (r) ...
```

Reply:
done